### PR TITLE
Adding member initializer list rule for C++ style guide.

### PR DESCRIPTION
**Pull request by abdouskamel**
**Fixes issue:**

#3232

**Changes:**

### CODE REVIEW

File: guides/coding_style/c++/README.md [hunk 1]
```diff
@@ -514,6 +514,20 @@ private:
 };
 ```
 
+### Member initializer list
+When declaring a constructor, use member initializer list to initialize class members.
```
Comment (arnavb) [inline guides/coding_style/c++/README.md]: use _a_ member initializer list.

File: guides/coding_style/c++/README.md [hunk 2]
```diff
@@ -514,6 +514,20 @@ private:
 };
 ```
 
+### Member initializer list
```
Comment (arnavb) [inline guides/coding_style/c++/README.md]: Capital I and L in the title.

Comment (abdouskamel) [reply guides/coding_style/c++/README.md]: Done. :)

File: guides/coding_style/c++/README.md (hunk 1 above)
Comment (abdouskamel) [reply guides/coding_style/c++/README.md]: Done. :)

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**